docs/how-to-play-9c-on-linux-and-mac.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Play 9c on x86 macOS / Linux using CI Artifacts (UNOFFICIAL SUPPORT)
+---
+# DISCLAMER
+THIS UNOFFICIAL METHOD DOES NOT GUARANTEE THE NORMAL OPERATION OF THE SOFTWARE (INCLUDING LAUNCHER AND GAME CLIENT) AND WE DOES NOT PROVIDE OFFICIAL TECHNICAL SUPPORT. OUR PRIMARY ISSUE PRIORITY IS ALWAYS WINDOWS FIRST. AND THESE TARGETS ARE USED FOR INTERNAL TEST ONLY. USE AT YOUR OWN RISKS. THIS EXPLANATION WILL PRESUME YOU KNOW WHAT YOU'RE DOING.
+
+# 1. Get Artifact
+
+## [1. Open 9c-launcher CircleCI Pipelines page](https://app.circleci.com/pipelines/github/planetarium/9c-launcher?branch=main)
+
+
+[Here.](https://app.circleci.com/pipelines/github/planetarium/9c-launcher?branch=main)
+
+## 2. Click Workflow "build", and click Succeed (Green) "build-{your OS}" 
+
+
+## 3. Click Artifact
+
+
+
+## 4. Download and Decompress Artifact
+
+
+
+# 2. Run on Linux
+
+
+After decompression, you path would look like this.
+
+Its permission is presetted so usually clicking Nine Chronicles launches launcher and will do it's job. (Downloading Game Client, Updating config.json, etc)
+
+In case of permission issue, `chmod +x ./Nine Chronicles` will do the trick.
+
+
+Now it works!
+
+config.json file is
+    - (on macOS) `~/Library/Application Support/Nine Chronicles/config.json`
+    - (on Linux) `~/.config/Nine Chronicles/config.json`
+
+And you can import private key using Libplanet.Tools, so called [`planet`](https://github.com/planetarium/libplanet/releases). or, you can mount windows and copy keystore folder under `%APPDATA%/planetarium/keystore` and put it on `~/.config/planetarium/keystore` I'll trust you would work this part out by yourself.
+
+# 2. Run on Intel Mac
+
+I don't have mac.
+
+But kind enough to left explanation.
```

Comment:
I don't know about our tone & manners of docs at this time :thinking: 
But I think joke-like expressions are used with care.
What do you think?

Reply:
I wrote this post originally and I want users to feel the same as before the translation.

Reply:
...but I'll change it to more formal tone.